.github/workflows/ci.yml

```
@@ -27,34 +26,71 @@ jobs:
+          name: screenshots-smoke
+          path: output/
+
+  regression-tests:
+    needs: smoke-tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)